content/javascript/concepts/operators/operators.md

```diff
@@ -175,3 +175,64 @@ let isDuck = (walksLikeADuck && talksLikeADuck) ? "It's a duck" : "It's not a du
 
 console.log(isDuck);
 ```
+
+The ternary operator can also perform multiple steps based on a certain condition, similar to the IF/ELSE condition statement.
+
+For example, the following IF/ELSE statement...
+
+```js
+let i = true;
+let a = 0;
+let b = 0;
+
+if (i == true) {
+    a++;
+    b++;
+} else {
+    a = 0;
+    b = 0;
+}
+```
+
+...can be translated into the following two-step ternary operator!
+
+```js
+let i = true;
+let a = 0;
+let b = 0;
+
+i == true ? (a++, b++) : (a = 0, b = 0);
+console.log('a = ', a, 'b = ', b);
+```
+
+The ternary operator can also perform as many steps as you prefer.
```

Comment:
```suggestion
The ternary operator can also perform as many steps as needed.
```

Reply:
Thank you @SSwiniarski for letting me know about these changes! I'm uploading them now.